tests/inside_worker_test/cast_to_float_or_null_test.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import sqlalchemy
+
+from tests.inside_worker_test.conftest import slow
+
+
+@pytest.fixture(params=[2, 2.2, 3.898986, "3.898986", "6", "0.2", 0.6])
+def valid_float_representation(request):
+    return request.param
+
+
+@pytest.fixture(params=["a2", "10b", "3.898986k", "3k.898986", "l6.9"])
```

Comment:
supporting `,` additionally to `.` I think is a good idea, but do we really want to support the thousand separators and similar awkwardness?